packages/victory-core/src/victory-accessibility-group/victory-accessibility-group.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+class VictoryAccessibilityGroup extends React.Component {
+  static displayName = "VictoryAccessibilityGroup";
+  static role = "container";
+  static propTypes = {
+    "aria-describedby": PropTypes.string,
+    "aria-label": PropTypes.string,
+    desc: PropTypes.string,
+    descId: PropTypes.string,
+    children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]),
+    className: PropTypes.string,
+    tabIndex: PropTypes.number
+  };
+
+  static defaultProps = {
+    className: "VictoryAccessibilityGroup"
+  };
+
+  render() {
+    const { desc, children, className, descId, tabIndex } = this.props;
+
+    return desc ? (
+      <g aria-label={this.props["aria-label"]} className={className} tabIndex={tabIndex}>
+        <desc aria-describedby={this.props["aria-describedby"]} id={descId}>
```

Comment:
I think `aria-describedby` belongs on the `g` tag. When there is a `desc`, `aria-describedby` should use the id of the `desc` tag. When there isn't a `desc`, it should use whatever user provided id string.

Reply:
ok cool, yeah I didn't get a lot out of looking up aria-describedby best practice, but in looking up reading on the `<desc>` tag i see its placement on `<desc>` is redundant. Will fix